amy/fiscal/tests/test_membership_members.py

```diff
@@ -310,6 +313,51 @@ def test_not_editing_noneditable_members_succeeds(self):
         )
         self.assertEqual(list(self.membership.organizations.all()), [self.org_alpha])
 
+    def test_mix_adding_removing_members_leaves_comment(self):
+        """Ensure a mixed-content formset for consortium membership members leaves
+        correct message in the comments."""
+        self.setUpMembership(consortium=True)
+        m1 = Member.objects.create(
+            organization=self.org_alpha,
+            membership=self.membership,
+            role=self.member_role,
+        )
+
+        data = {
```

Comment:
Could you add a comment where beta is added and alpha is deleted? I was looking through this and was not sure how those actions were happening.

Reply:
Somehow I missed your comment. I added comments to the form content below, hopefully this helps.

Reply:
perfect thank you!